src/redux/actions/createAction.js

```diff
@@ -0,0 +1,14 @@
+export const createAction = actionName => {
+  const action = payload => ({ type: actionName, payload });
+  action.toString = () => actionName;
+  return action;
+};
+
+export const createDispatcher = (func, params, success, error) => async dispatch => {
+  try {
+    const data = await func(params);
+    dispatch(success(data));
+  } catch ({ errors }) {
+    dispatch(error(errors));
+  }
+};
```

Comment:
Nice abstraction!

If you want to support multiple params you can do something like this:
```suggestion
export const createDispatcher = (func, success, error, ...params) => async dispatch => {
  try {
    const data = await func(...params);
    dispatch(success(data));
  } catch ({ errors }) {
    dispatch(error(errors));
  }
};
```